sidebars/core.js

```diff
@@ -29,7 +29,6 @@ const Sidebar = [
     label: 'Concepts',
     type: 'category',
     items: [
-      'demo',
```

Comment:
Demo needs to be in its own section under reference

Reply:
@jjwill  It is, it's just called something different (v2-docs-guide)

Reply:
Sweet, thanks. Try to show all of that in the same PR next time.

Reply:
It is. Check sidebars/components.js.. Linking to the commit instead of the whole pr because now I've merged in some other commits: https://github.com/dfuzr/libra-website/pull/114/commits/7c2c4531b10d54cac3cca908e3b4be1e906b7294#diff-2a24fa5de3c32ee0840420ff2ed5005aR29-R36